lib/src/components/mixins/single_child_particle.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:ui';
 
-import '../../particle.dart';
+import '../../particles/particle.dart';
```

Comment:
Couldn't this just import `lib/particles.dart` or is that not according to the standard?

Reply:
It imports what needs to be imported, it doesn't need any of the particle implementations.